docs/contributing/index.md

```diff
@@ -13,91 +9,92 @@ Ready to contribute? Here's how to set up `scvi-tools` for local development.
 
 1.  Fork the `scvi-tools` repo on GitHub.
 
-2.  Clone your fork locally:
+1.  Clone your fork locally:
 
     ```
     # Clone your fork of the repository (substitute in your username)
     git clone https://github.com/{your-username}/scvi-tools.git
     # Enter the cloned repository
     cd scvi-tools
     # Add our repository as a remote
-    git remote add upstream https://github.com/yoseflab/scvi-tools.git
-    # git branch --set-upstream-to "upstream/master"
+    git remote add upstream https://github.com/scverse/scvi-tools.git
+    # git branch --set-upstream-to "upstream/main"
     ```
 
-3.  Install your local copy into a virtualenv (or conda environment):
+1.  Setup a virtual environment:
 
     ```
     # If you have pyenv-virtualenv
     pyenv virtualenv scvi-tools-dev
     pyenv activate scvi-tools-dev
-    # If you have conda (omit the python parameter if you already have the relevant python version installed)
-    conda create -n scvi-tools-dev python=3.8.8 # or any python >3.7 that is available (conda search python)
-    conda activate scvi-tools-dev
-    # Enter the cloned repository and install the package in editable mode
-    cd scvi-tools
-    pip install -e ".[dev,docs,tutorials]"
+    # If you have mamba (omit the python parameter if you already have the relevant python version installed)
+    mamba create -n scvi-tools-dev python=3.8.8 # or any python >3.7 that is available (mamba search python)
+    mamba activate scvi-tools-dev
     ```
 
-    To confirm that scvi-tools was successfully installed:
+1.  (Optional) Install GPU versions of PyTorch and jax:
+
+    Using mamba (recommended):
 
     ```
-    # This should find the package. Note that other metadata (such as Version, Summary, etc.) might be missing. This
-    # is expected because we use poetry instead of setup-tools. On a non-editable install, these would be populated.
-    pip show scvi-tools
+    mamba install pytorch torchvision torchaudio cudatoolkit=11.6 -c pytorch -c conda-forge
+    mamba install jax jaxlib cuda-nvcc -c conda-forge -c nvidiai
```

Comment:
Should we add a note here that `cuda-nvcc` should match the CUDA version on their device?

Reply:
I think it's complicated, because I believe you can have no global device cuda and install it only with conda. So I rather not enumerate all the possibilities here

Reply:
Ok I see, sounds good